config/mainCfg_TauTau_UL18.cfg

```diff
@@ -3,58 +3,40 @@
-variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, nbjetscand, njets
-#selections = baseline, baselineNoB, s1b1jresolvedInvMcut, s2b0jresolvedInvMcut, sboostedLLInvMcut#, baselineNewTrg, s1b1jresolvedMcut, s2b0jresolvedMcut, sboostedLLMcut, s1b1jresolvedInvMcut, s2b0jresolvedInvMcut, sboostedLLInvMcut, s1b1jresolvedMcutNewTrg, s2b0jresolvedMcutNewTrg, sboostedLLMcutNewTrg, s1b1jresolvedInvMcutNewTrg, s2b0jresolvedInvMcutNewTrg, sboostedLLInvMcutNewTrg#, GGFclass, VBFclass, ttHclass, TTclass, DYclass
+backgrounds = DY_NLO_incl_stitch, DY_NLO_0J, DY_NLO_1J, DY_NLO_2J, DY_NLO_Pt50To100, DY_NLO_Pt100To250, DY_NLO_Pt250To400, DY_NLO_Pt400To650, DY_NLO_Pt650ToInf, TTfullyHad, TTfullyLep, TTsemiLep, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, TWtop, TWantitop, singleTop_top, singleTop_antitop, ZH_HTauTau, WplusHTauTau, WminusHTauTau, WW, WZ, ttHJetTononBB, ttHJetToBB, ttHJetToTauTau, TTWW, TTWZ, TTZZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu, ggHTauTau #DY_NLO_incl
 
-#selections = baseline, baseline_noMCw, baseline_noPUw, baseline_noL1w, baseline_noTrigw, baseline_noDYw, baseline_noprescalew, baseline_noPUjetw, baseline_nobtagw, baseline_notauidw
-#selections = baseline, baseline_noTrigw, baseline_notauidw, baselineQCD, baselineQCD_noTrigw, baselineQCD_notauidw
-selections = baseline#, baseline_DM#, baselineQCD_loweriso, baselineQCD_loweriso_nob, baselineNoB, baseline_40pt50, baseline_50pt70, baseline_70pt100, baseline_100ptInf, baselineNoB_40pt50, baselineNoB_50pt70, baselineNoB_70pt100, baselineNoB_100ptInf, baselineQCD_loweriso_nob_40pt50, baselineQCD_loweriso_nob_50pt70, baselineQCD_loweriso_nob_70pt100, baselineQCD_loweriso_nob_100ptInf,  baselineQCD_loweriso_40pt50, baselineQCD_loweriso_50pt70, baselineQCD_loweriso_70pt100, baselineQCD_loweriso_100ptInf
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, dau1_phi, dau2_phi, ditau_deltaR, bjet1_pt, bjet1_bID_deepFlavor, bjet2_pt, bjet2_bID_deepFlavor, bjet1_eta, bjet2_eta, tauH_mass, tauH_SVFIT_mass, tauH_pt, bH_mass, bH_pt, HH_mass, HH_pt, HHKin_mass, dau1_iso, dau2_iso, dib_deltaR, btau_deltaRmin, btau_deltaRmax, HH_deltaR, dau1_deepTauVsMu, dau1_deepTauVsEle, dau1_deepTauVsJet, dau2_deepTauVsMu, dau2_deepTauVsEle, dau2_deepTauVsJet, HT20Full #BDToutSM_kl_1
 
+selections = baseline, s1b1jresolvedInvMcut, s2b0jresolvedInvMcut, sboostedLLInvMcut#, baselineNewTrg, s1b1jresolvedMcut, s2b0jresolvedMcut, sboostedLLMcut, s1b1jresolvedInvMcut, s2b0jresolvedInvMcut, sboostedLLInvMcut, s1b1jresolvedMcutNewTrg, s2b0jresolvedMcutNewTrg, sboostedLLMcutNewTrg, s1b1jresolvedInvMcutNewTrg, s2b0jresolvedInvMcutNewTrg, sboostedLLInvMcutNewTrg#, GGFclass, VBFclass, ttHclass, TTclass, DYclass
```

Comment:
Could you put back the actual baseline categories in the configs before we merge ?

Reply:
Done.